lib/auth/init.go

```diff
@@ -1048,12 +1048,23 @@ type PresetUsers interface {
 	UpsertUser(ctx context.Context, user types.User) (types.User, error)
 }
 
+// getPresetUsers returns a list of all users roles expected to be available on
```

Comment:
should be just users?
```suggestion
// getPresetUsers returns a list of all preset users expected to be available on
```